honeybee_vtk/cli/export.py

```diff
@@ -73,10 +74,20 @@ def export():
     help='Validate simulation data before loading on the model. This is recommended'
     ' when using this command locally.', show_default=True
 )
+@click.option(
+    '--grid', is_flag=True, default=False, help='Boolean to decide whether to export'
+    ' the images of a whole model or only the grids. Set it to True to export the grids.',
+    show_default=True
+)
```

Comment:
To make this recipe friendly we should have two values for the flag.
```suggestion
@click.option(
    '--grid/--model', is_flag=True, default=False, help='Boolean to decide whether to export'
    ' the images of a whole model or only the grids. Set it to True to export the grids.',
    show_default=True
)
```